doc/bevindingen-fase-3.md

```diff
@@ -45,18 +45,52 @@ Zie ook [Topsector-Logistiek/Apache-Pulsar-Auth-Plugin](https://github.com/Topse
 
-Wat hier lastig is, is dat er geen mechanisme is om te achterhalen waar het token endpoint voor een gegeven URL te vinden is.  Hiervoor hebben we gekozen voor een `WWW-Authenticate` header in het `401 Unauthorized` response waarin beschreven staat waar het  endpoint is en wat de EORI van de server is (nodig om een  `client_assertion` te kunnen opstellen).
+Voorbeeld `WWW-Authenticate` header:
+
+```
+WWW-Authenticate Bearer scope="iSHARE" server_eori=*EORI-PRODUCER* server_token_endpoint=*TOKEN-URL-PRODUCER*
+```
 
 # Conclusie
 
-TODO
+Het gebruik van een *event broker* in deze opzet heeft de volgende gevolgen:
+
+- de event broker wordt weer een extra partij in het afsprakenstelsel
+
+  De event broker moet worden ge-onboard, een iSHARE token-endpoint implementeren en de *associate en autorisatie registers* bevragen om de toegang te verifiëren.  Dit levert extra druk op deze componenten en introduceert foutpunten welke stabiele werking kunnen belemmeren.
+  
+- er is een ontdekkingsmechanisme (discovery) nodig om event brokers te vinden
+
+  Op dit moment is er geen methode om aan te geven waar de event broker voor een transportopdracht gevonden kan worden.  Dit zou onderdeel gemaakt kunnen worden van de transportopdracht data die gestuurd wordt vanuit het ERP naar de betrokken partijen.  Het is ook mogelijk hiervoor DNS discovery in te zetten.  Dit vergt meer onderzoek.
+
+- eigenaarschap van en toegang tot topics en events in deze opstelling is niet op de proef gesteld
+
+  Het versturen van slechts een gate-out event door het DC geeft geen goed beeld van de rollen die belegd worden in de hele levenscyclus van een transportopdracht. Het onderwerp *van wie wat mag zien* is een belangrijk onderwerp in de BDI en komt nu in deze fase duidelijk naar voren. Deze aanpak voldoet alleen voor dit ene gate-out event, maar zodra er andere events bij gaan komen, gaat afscherming een rol spelen en *data bij de bron* is hier geen oplossing voor omdat het feit dat een DC een event verstuurt al veel te veel prijsgeeft.
+
+De toepassing van *data bij de bron* in deze opzet heeft de volgende gevolgen:
+
+- een producent van events moet een publiek bereikbaar end-point aanbieden om events op te halen
+
+- een producent moet een authenticatie mechanisme implementeren om dit end-point te beschermen
+
+- een producent moet ergens kunnen verifiëren of een partij die een event komt ophalen daar ook recht op heeft
+
+  In deze opstelling lijkt het voor de hand te liggen dat zowel de opdrachtgever (verlader) als het transportbedrijf dat bij het DC komt deze gegevens op mag halen. Maar zodra er meer partijen betrokken worden in de opdracht wordt dit veel ingewikkelder.  Denk hierbij aan track-and-trace gegevens voor afnemers die de verlader opdracht hebben gegeven, het DC heeft hier geen weet van.
```

Comment:
boven bij "Delegation-Evidence bij uitbesteding aan derde" zeg je juist dat hier een oplossing voor is maar dat die niks toevoegd. Ik zou daarnaar verwijzen en het op een of andere manier consistent maken